Notifications replay after App Center initialized (#392)
Microsoft/AppCenter-SDK-Unity: CHANGELOG.md

# App Center SDK for Unity Change Log

## Version 2.6.1 (Under development)

### App Center Push 

#### iOS/UWP

* **[Bug fix]** Fix missed push notification window if notification was received after the application closed.

> 

Reviewer: ```suggestion

___

```

# App Center SDK for Unity Change Log

## Version 2.6.1 (Under development)

### App Center Push 

#### iOS/UWP

> * **[Bug fix]** Fix missed push notification window if notification was received after the application closed.

Reviewer 2: I don't understand what this window is about, our SDK does not provide any UI for push.

Reviewer: window -> event

Reviewer 2: ```suggestion
* **[Bug fix]** Fix triggering the notification event when restarting the application from a notification.
```
Or something along those lines.

* **[Fix]** Fix an issue where Push could not be started because `StartPush` method was stripped for a high managed stripping level.

> #### iOS/UWP

Reviewer 2: I see an Android patch too in the PR which seems to change behavior since we move the replay method to start, I'm curious why it's not mentioned here.

Reviewer 3: As I remember android part worked before and still works.

Reviewer 2: So what was that change about then?

Reviewer 3: That's change for iOS (as well as the bug itself)

Reviewer 4: Are you saying that `Push.ReplayPushNotificationsIfWaiting();` was moved to `PostInitialize` in Android just to align it with the iOS implementation, but it worked fine when it was in the `Initialize` function?

Reviewer 2: I am curious why android was changed as well but not mentioned in changelog. What is the rationale of that change.
